packages/koa-shopify-auth/src/auth/client/request-storage-access.ts

```diff
@@ -10,7 +10,7 @@ const requestStorageAccess = (shop: string, prefix = '') => {
           doesNotHaveStorageAccessUrl: "${prefix}/auth/enable_cookies?shop=${encodeURIComponent(
     shop,
   )}",
-          appTargetUrl: "/?shop=${encodeURIComponent(shop)}"
+          appTargetUrl: "${prefix}?shop=${encodeURIComponent(shop)}"
```

Comment:
I believe we'll still need the `/` at the beginning. Can we update the default `prefix` in the function argument to be `'/'` instead of `''`?. Users who set a prefix will need to make sure the use a `/` (eg. `/my-prefix`)

Reply:
👆

Reply:
You are right, sorry. I've updated PR, so it will use '/' path in case there is no prefix setup. Thanks a lot for reviewing this!